app/Resources/views/admin/blog/show.html.twig

```diff
@@ -38,12 +38,7 @@
     </div>
 
     <div class="section">
-        {{ include('admin/blog/_form.html.twig', {
-            form: delete_form,
-            button_label: 'action.delete_post'|trans,
-            button_css: 'btn btn-lg btn-block btn-danger',
-            show_confirmation: true,
-        }, with_context = false) }}
+        {{ include('admin/blog/_delete_form.html.twig', { post: post}, with_context = false) }}
```

Comment:
Missing space to end `{ post: post }` or remove extra space to beginning `{post: post}` ? same in other files.